cli/cmd/project/clone.go

```diff
@@ -0,0 +1,153 @@
+package project
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	githttp "github.com/go-git/go-git/v5/plumbing/transport/http"
+	"github.com/rilldata/rill/cli/cmd/env"
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/dotrillcloud"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func CloneCmd(ch *cmdutil.Helper) *cobra.Command {
+	var path string
+
+	cloneCmd := &cobra.Command{
+		Use:   "clone [<project-name>]",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Square parentheses means optional:

```suggestion
		Use:   "clone <project-name>",
		Args:  cobra.ExactArgs(1),
```